apigee_sync_datascope_cache.go

```diff
@@ -0,0 +1,93 @@
+package apidApigeeSync
```

Comment:
I'm not sure why you named the file this: apigee_sync_datascope_cache.go? Wouldn't datascope_cache.go be sufficient?